plugin/storage/es/spanstore/reader_test.go

```diff
@@ -145,30 +145,30 @@ func TestSpanReaderIndices(t *testing.T) {
 	date := time.Now()
 	dateFormat := date.UTC().Format("2006-01-02")
 	testCases := []struct {
-		index string
-		params  SpanReaderParams
```

Comment:
The formatting changes were made automatically by my editor and should match the other files in the project. Let me know if I should revert them.

Reply:
cannot be the formatting verified during the build. There are probably other files which does not match setting in your editor.

I would prefer to keep only changes related to the subject.

Reply:
@objectiser also expressed that via IRC, and I'd normally undo such changes, but these ones do make sense to me. For instance, it would be expected that `index string` would be aligned wiht `params SpanReaderParams` and that was *not* the case before this PR. As this is a small PR, and the "after" state is better than the "before", I decided to leave the formatting changes there.

If it's still desirable to revert those changes, I can certainly do that.

Reply:
that looks like a standard go fmt behavior, is it not running on this file during the build?  I remember we used to have an issue where lints were not running on all Go files.

Reply:
Turns out we indeed were excluding test files from linting. Fix here: #1709. Once we merge that, this PR won't need formatting diffs